www/docs/devel/collab.md

```diff
@@ -0,0 +1,37 @@
+# On Collaborative Development
```

Comment:
delete all of this, since it is redundant with things in our other docs directory.